_toDo/costumes2/data/heroes/4star_KirilCostume2.yml

```diff
@@ -0,0 +1,53 @@
+- name: Kiril
+  class: Wizard
+  source: Season1
+  speed: Average
+  power: 647
+  attack: 656
+  defense: 673
+  health: 1034
+  skill: Blessed Brew
+  effects:
+    - Recovers 28% health for all allies
+    - All allies get +30% attack for 3 turns
+    - All allies get +30% defense for 3 turns
+  types:
+    - Healer
+    - Attack Buff
+    - Defense Buff
+  family: Classic
+  image: https://i.imgur.com/OnjIJvT.jpg
+  costume:
+    class: Sorcerer
+    power: 682
+    attack: 702
+    defense: 680
+    health: 1153
+    skill: Blessed Solo
+    effects:
+      - Recovers 28% health for all allies
+      - All enemies get -34% attack for 3 turns
+      - All enemies get -34% defense for 3 turns
+    types:
+      - Healer
+      - Attack Debuff
+      - Defense Debuff
+    family: Classic
+    image: https://i.imgur.com/BwPG5qR.jpg
+costume2:
```

Comment:
Be careful with the indentations.  Yaml is very picky, so stuff like this will throw an error.

Reply:
Will do all the changes of course. And good point with the indentations, looks like a copy & paste error after so many files.

The costume bonus I cannot change yet though since i renamed "default" into "classic", just to be prepared in case SGG decides to put up some new costume format again. This would mean that the current bonus calculation would not be shown anymore.

Glad to be of help! :)

Greetings